packages/meditrak-app/app/entityMenu/EntityList.js

```diff
@@ -96,29 +105,29 @@ export class EntityList extends PureComponent {
       return;
     }
 
-    const { entities } = this.props;
-    const lowerCaseSearchTerm = searchTerm.toLowerCase();
-
-    const searchResults = entities
-      .filter(
-        ({ name, parentName }) =>
-          name.toLowerCase().includes(lowerCaseSearchTerm) ||
-          (parentName && parentName.toLowerCase().includes(lowerCaseSearchTerm)),
-      )
-      .sort(({ name: a }, { name: b }) => {
-        // Send entity names that start with the search term to the top.
-        const checkMatchesAtStart = name => name.toLowerCase().startsWith(lowerCaseSearchTerm);
-        if (checkMatchesAtStart(a) !== checkMatchesAtStart(b)) {
-          return checkMatchesAtStart(a) ? -1 : 1;
-        }
+    const { entities } = this.state;
 
-        // Sort alphabetically
-        return a.localeCompare(b);
+    const primarySearchResults = entities.filtered(`name BEGINSWITH[c] $0`, searchTerm);
```

Comment:
Are we losing the alphabetical sorting that the previous results used to have? Or are they sorted alphabetically by the db?

Reply:
Great point, we're losing it!

Reply:
Actually just checked and that is still retained - the sorting is done a little deeper and persists on the realm results object that this filters.